week2/week2-task4/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+
+var time = new JupiterTime(2, 20);
+PrintTime(time);
+
+var timeIn20Minutes = time.AddMinutes(150);
+PrintTime(timeIn20Minutes);
+
+void PrintTime(JupiterTime time)
+{
+    Console.WriteLine($"The time is {time.Hours:00} : {time.Minutes:00}");
+}
+public class JupiterTime
+{
+
+
```

Comment:
Try to avoid unnecessary empty lines, unless they are used for separating logic or functionality